src/main/scala/rx/lang/scala/package.scala

```diff
@@ -24,11 +25,11 @@ import _root_.scala.util.{Try, Success, Failure}
  */
 package object scala {
 
-  /**
-   * Placeholder for extension methods into Observable[T] from other types
-   */
-  implicit class ObservableExtensions[T](val source: Iterable[T]) extends AnyVal {
-      def toObservable: Observable[T] = {  Observable.from(source) }
+  @deprecated("Use IterableToObservable", "0.26.5")
+  type ObservableExtensions[T] = IterableToObservable[T]
+
+  implicit class IterableToObservable[T](val iterable: Iterable[T]) extends AnyVal {
```

Comment:
This actually will break binary compatibility. It's better to keep the class name and the parameter name because I think this implicit may be one of the most popular APIs.